.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow runs ECM's nightly tests, which currently only includes performance tests
+
+name: Nightly Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 7 * * *' # run at 7 AM UTC, 2 AM ET.
+
+jobs:
+  performance-tests:
+    runs-on: [self-hosted]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check out terra-helmfile version directories
+        uses: ./.github/actions/checkout-helm-versions
+        with:
+          token: ${{ secrets.BROADBOT_TOKEN }}
+          path: integration/terra-helmfile
```

Comment:
We now have an alternative way of getting the `terra-helmfile` instead of using the `checkout-helm-versions`. It's up to you but here's the alternative commands that achieve the same results. You can take a look at the `run-perf-tests-and-publish.yml` workflow this [PR](https://github.com/DataBiosphere/terra-data-catalog/pull/42/files).

```
      - name: Alternative way of getting terra-helmfile repo version files
        run: |
          curl -H 'Authorization: token ${{ secrets.BROADBOT_TOKEN }}' \
            -H 'Accept: application/vnd.github.v3.raw' \
            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/versions/app/dev.yaml \
            --create-dirs -o "integration/terra-helmfile/versions/app/dev.yaml"
          curl -H 'Authorization: token ${{ secrets.BROADBOT_TOKEN }}' \
            -H 'Accept: application/vnd.github.v3.raw' \
            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/environments/live/perf.yaml \
            --create-dirs -o "integration/terra-helmfile/environments/live/perf.yaml"
```